diff --git a/components/VirtualList.js b/components/VirtualList.js
index 2e9d6ba25e0692493eaffdde6867b0fb134613b9..157f5690b655c2e960927cb7a3a40cf3280f2a20 100644
--- a/components/VirtualList.js
+++ b/components/VirtualList.js
@@ -1,6 +1,6 @@
 'use client';
 import { jsx as _jsx } from "react/jsx-runtime";
-import { List } from 'react-window';
+import { FixedSizeList as List } from 'react-window';
 export function VirtualList({ itemCount, itemSize = 36, height, width = '100%', children: renderRow, overscanCount = 5, }) {
     return (_jsx(List, { rowComponent: (p) => renderRow({ index: p.index, style: p.style }), rowCount: itemCount, rowHeight: itemSize, rowProps: undefined, style: { height: Math.min(height, itemCount * itemSize), width }, overscanCount: overscanCount }));
 }
